src/contributing.md

```diff
@@ -4,9 +4,7 @@ The [Exonum core project][exonum] and [other public Exonum repositories][exonum-
 operate an open contributor model where anyone is welcome to contribute
 towards development in the form of peer review, testing and patches.
 This document explains the practical process and guidelines for contributing.
-
-!!! note
-    This contribution guide is partially derived from [the Bitcoin contribution guide][btc-contrib].
+It is partially derived from [the Bitcoin contribution guide][btc-contrib].
```

Comment:
I'd use "This contribution guide" or "This guide" instead of "It"; imo, "it" may be a little confusing since paragraphs should make sense in insolation. Alternatively, you may move the sentence to the previous paragraph, although I would advise not to do this: the first para of the page is automatically used as the page description in meta, so bloating it would not be wise.

Reply:
>Alternatively, you may move the sentence to the previous paragraph

It is already in the 1st paragraph, see the formatted document. And its a credit, so it possibly should have good visibility.

Reply:
Oh sorry, missed that (didn't look at the rendered docs). Yeah, the sentence after your edits looks completely fine.

Reply:
np! Markdown authoring is very nice, but I get myself sometimes confused from the render-discrepancies.